<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCorrect</class>
 <widget class="QDialog" name="DialogCorrect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>281</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Free compart</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Data</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLineEdit" name="freecompartEdit"/>
    </item>
    <item row="6" column="6">
     <widget class="QLabel" name="reservedAlarm">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="numberAlarm">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QLabel" name="destinationAlarm">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="dataAlarm">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="reservedEdit"/>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="dataEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Destination</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>compart</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Free reserved</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QLabel" name="freereservedAlarm">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="compartEdit"/>
    </item>
    <item row="7" column="6">
     <widget class="QLabel" name="freecompartAlarm">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QLabel" name="compartAlarm">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QLineEdit" name="freereservedEdit"/>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="destinationEdit"/>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="timeEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>reserved</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="numberEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>number</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="timeAlarm">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
